servers/quarkus-server/src/test/java/org/projectnessie/server/ITNativeRestApiMongo.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server;
+
+import io.quarkus.test.junit.NativeImageTest;
+import io.quarkus.test.junit.TestProfile;
+import java.net.URI;
+import org.junit.jupiter.api.BeforeEach;
+import org.projectnessie.jaxrs.AbstractTestRest;
+import org.projectnessie.server.profiles.QuarkusNativeProfileMongo;
+
+@NativeImageTest
+@TestProfile(QuarkusNativeProfileMongo.class)
+public class ITNativeRestApiMongo extends AbstractTestRest {
+
+  @Override
+  @BeforeEach
+  public void setUp() throws Exception {
```

Comment:
Feels like we should introduce some `AbstractQuarkusTestRest` class, because the `setUp` method's duplicated a lot.

Reply:
Sure. I'll do this is a smaller PR presently

Reply:
#2164